en/docs/develop-components/develop-web-applications/develop-web-applications-locally-with-managed-authentication.md

```diff
@@ -0,0 +1,76 @@
+# Develop Web Applications Locally with Choreo’s Managed Authentication
+
+Choreo’s  Managed Authentication capability exposes a set of backend for frontend (BFF) endpoints, facilitating authentication and authorization for web applications. These endpoints are readily available for single-page web applications deployed on Choreo.
+
+As a web application developer, there may be instances where it becomes necessary to run the application on your personal workstation. In such situations, the expectation is for authentication and authorization during local development to work in the same manner as it would when the application is deployed to Choreo, eliminating the need to modify the source code.
+
+Local development in Choreo provides the same capabilities as developing the web application locally.
```

Comment:
There seems to be a disconnect with the previous paragraph and this. WDYT @Nashaath?

Reply:
Shall we remove this sentence and perhaps have a para describing Choreo's local development capability? Something along the lines of 
"Choreo's local development capability ensures a seamless local application development experience where the authentication and authorization process aligns with Choreo’s deployment environment. This facilitates a smoother transition from development to deployment, improving overall workflow efficiency."